### ISSUE TITLE
Why is the model useless？

### ISSUE THREAD

**Issue by AmorFatiliwenjie** [pengmiao-usc/TransFetch#2]
I followed the process in readme, but the result of prefetching the generated file to the simulator is equivalent to no prefetching. What is the reason for this? Do you need to modify the training parameters? How to set up the training set, validation set, testing set, and skip?

**Comment by AmorFatiliwenjie** [comment]
I've fixed this issue, but the accuracy of the results is low
